src/dashboard/CreatorCard.vue

```diff
@@ -17,19 +16,20 @@ b-card(class="mb-2" no-body)
         b-input-group(prepend="ETH Address", size="sm")
           b-form-input(v-model="address")
 
-      p(v-if="creator.info").text-small {{ creator.info }}
+      p(v-if="creator.info").text-small
+        | {{ creator.info }}
 
       b-button(size="sm", :variant="'outline-secondary'", v-on:click="showDetails = !showDetails")
-        | #[span(v-if="!showDetails") Show] #[span(v-else) Hide] details 
-        font-awesome-icon(icon="info-circle")
+        | #[font-awesome-icon(icon="info-circle")] #[span(v-if="!showDetails") Show] #[span(v-else) Hide] details
       b-button(v-if="creator.edited", v-on:click='$emit("edit")', variant='outline-secondary', size='sm').float-right
-        | Edit 
-        font-awesome-icon(icon="edit")
+        | #[font-awesome-icon(icon="edit")] Edit
 
       table.table.table-sm.table-hover.mt-3.mb-0(v-if="showDetails")
         tr
           th Page
           th Duration
+        tr(v-if="activities.length == 0" colspan="0")
+          td No activity found for creator
```

Comment:
This looks a bit weird (the bar)
![image](https://user-images.githubusercontent.com/1427063/42219734-34ac82ac-7ecd-11e8-85bd-6785824f57c5.png)